app/models/User.py

```diff
@@ -118,8 +119,9 @@ def get_profile_picture(self, size=50):
         if self.image:
             return self.image.url()
         if not self.image_url:
-            return ('https://lh6.googleusercontent.com/-K9HZ5Z5vOU8/'
-                    'AAAAAAAAAAI/AAAAAAAAAAA/yRoMtBSXoxQ/s48-c/photo.jpg')
+            from app import app
```

Comment:
*sigh*

So right now we're using the "create_app" pattern, where we initialize `app` to `None` [here](https://github.com/adicu/adi-website/blob/master/app/__init__.py#L21) and then run  [`create_app()`](https://github.com/adicu/adi-website/blob/master/app/__init__.py#L27), which populates it.

The problem with this pattern is that if you have modules that depend on a non-`None` `app` and those modules are imported in the process of creating `app` in `create_app` then you have a recursive problem: you can't create the app with out importing, you can't import without a created app.

The solution is to `from app import app` as late as possible during the runtime (in this case, inside the function) so that we can guarantee that `app` is not `None` when we do.

It's ugly, but the best solution I've found